src/renderers/dom/server/ReactServerUpdateQueue.js

```diff
@@ -0,0 +1,115 @@
+      ReactNoopUpdateQueue.isMounted(publicInstance);
+  },
+
+  /**
+   * Enqueue a callback that will be executed after all the pending updates
+   * have processed.
+   *
+   * @param {ReactClass} publicInstance The instance to use as `this` context.
+   * @param {?function} callback Called after state is updated.
+   * @internal
+   */
+  enqueueCallback: function(publicInstance, callback, callerName) {
+    return this.transaction.isInTransaction() ?
+      ReactUpdateQueue.enqueueCallback(publicInstance, callback, callerName) :
+      ReactNoopUpdateQueue.enqueueCallback(publicInstance, callback, callerName);
+  },
+
+  /**
+   * Forces an update. This should only be invoked when it is known with
+   * certainty that we are **not** in a DOM transaction.
+   *
+   * You may want to call this when you know that some deeper aspect of the
+   * component's state has changed but `setState` was not called.
+   *
+   * This will not invoke `shouldComponentUpdate`, but it will invoke
+   * `componentWillUpdate` and `componentDidUpdate`.
+   *
+   * @param {ReactClass} publicInstance The instance that should rerender.
+   * @internal
+   */
+  enqueueForceUpdate: function(publicInstance) {
+    return this.transaction.isInTransaction() ?
+      ReactUpdateQueue.enqueueForceUpdate(publicInstance) :
+      ReactNoopUpdateQueue.enqueueForceUpdate(publicInstance);
+  },
+
+  /**
+   * Replaces all of the state. Always use this or `setState` to mutate state.
+   * You should treat `this.state` as immutable.
+   *
+   * There is no guarantee that `this.state` will be immediately updated, so
+   * accessing `this.state` after calling this method may return the old value.
+   *
+   * @param {ReactClass} publicInstance The instance that should rerender.
+   * @param {object} completeState Next state.
+   * @internal
+   */
+  enqueueReplaceState: function(publicInstance, completeState) {
+    return this.transaction.isInTransaction() ?
+      ReactUpdateQueue.enqueueReplaceState(publicInstance, completeState) :
+      ReactNoopUpdateQueue.enqueueReplaceState(publicInstance, completeState);
+  },
+
+  /**
+   * Sets a subset of the state. This only exists because _pendingState is
+   * internal. This provides a merging strategy that is not available to deep
+   * properties which is confusing. TODO: Expose pendingState or don't use it
+   * during the merge.
+   *
+   * @param {ReactClass} publicInstance The instance that should rerender.
+   * @param {object} partialState Next partial state to be merged with state.
+   * @internal
+   */
+  enqueueSetState: function(publicInstance, partialState) {
+    return this.transaction.isInTransaction() ?
+      ReactUpdateQueue.enqueueSetState(publicInstance, partialState) :
+      ReactNoopUpdateQueue.enqueueSetState(publicInstance, partialState);
+  },
+};
+
+Object.assign(
+  ReactServerUpdateQueue.prototype,
+  Mixin
+);
+
+module.exports = ReactServerUpdateQueue;
```

Comment:
Yea, I’m not 100% sure (we don’t seem to use classes yet) but this could be the first one 😉 . I think `classes` is in `babelrc` so it should be good. But check compiled output to make sure.